### PR TITLE
apply fonts

### PR DESCRIPTION
**Pull request by niehaus-alexander**
Since Poetsen One and Concert One look way more similar than we anticipated, we decided to use just Concert One throughout the whole app. We are aware this deviates slightly from our ticket.

### CODE REVIEW

File: styles.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=Concert+One&display=swap');
```
Comment (SabinePfanner) [inline styles.js]: You should use ```next/font``` to implement Google Fonts in a save and compliant way. Check handout of session "React Styled Components 1":  https://github.com/neuefische/hh-web-24-4/blob/main/sessions/react-styled-components-1/react-styled-components-1.md

**Review by SabinePfanner — CHANGES_REQUESTED** [review]
Check session "React Styled Components" for implementing Google Fonts via next/font

File: styles.js (hunk 1 above)
Comment (niehaus-alexander) [reply styles.js]: Done, thank you :)

File: pages/_document.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ export default class MyDocument extends Document {
   render() {
     return (
       <Html lang="en">
-        <Head />
+        <Head></Head>
```
Comment (SwethaJanardhana) [inline pages/_document.js:29]: since  `<Head></Head>` doesn't have any content and children, it can be a self closing tag.

**Review by SwethaJanardhana — COMMENTED** [review]
LGTM, left a tip

**Pull request merged**